install.sh

```diff
@@ -1,89 +1,123 @@
-#!/usr/bin/bash
-
-# Copy fanctrl.py to /usr/local/bin and creates a service to run it
-# Adapted from https://gist.github.com/ahmedsadman/2c1f118a02190c868b33c9c71835d706
+#!/bin/bash
 
 if [ "$EUID" -ne 0 ]
-  then echo "Please run as root"
-  exit
+  then echo "This program requires root permissions"
+  exit 1
 fi
 
-SERVICE_NAME="fw-fanctrl"
-
-if [ "$1" = "remove" ]; then
-
-    sudo systemctl stop ${SERVICE_NAME//'.service'/} # remove the extension
-    sudo systemctl disable ${SERVICE_NAME//'.service'/}
-    rm /usr/local/bin/fw-fanctrl
+LOGNAME="$(logname)"
+USER="$LOGNAME"
+HOME="$(eval "~$USER")"
```

Comment:
For some reasons, this doesn't work for me.
Bash 5.1.16.
It outputs `./install.sh: line 10: /home/myusername: Is a directory`
`HOME=~$USER` seems to work

Reply:
I will check that rn, thanks for the feedback

Reply:
`HOME="$(getent passwd $USER | awk -F: '{print $6}')"` does work (courtesy of chatGPT)

Reply:
meh, I'm not a fan of using external tools for this type of things if it can be done in a conventional way
I will dig further about the ~$USER resolution and keep you informed

Reply:
ok, last chance here, does `HOME="$(su -c 'echo ~' "$USER")"` solve your problem?

Reply:
Damns, you're here already, didn't see you answered when I posted the chatGPT thingy
`HOME="$(eval echo "~$USER")"` works fine, thanks :+1:

Reply:
nice, lets go for it then